phys/module_physics_init.F

```diff
@@ -2622,7 +2622,7 @@ SUBROUTINE bl_init(STEPBL,BLDT,DT,RUBLTEN,RVBLTEN,RTHBLTEN,  &
                                                                 ) !Optional oml
 !--------------------------------------------------------------------
    USE module_sf_sfclay
-   USE module_sf_sfclayrev
+   USE sf_sfclayrev
```

Comment:
The depend.common line for this under module_physics_init should change to this new module

Reply:
It is already there. But module_sf_sfclayrev.o could be removed.